src/ucp/proto/proto_perf.h

```diff
@@ -28,13 +28,47 @@ typedef enum {
     UCP_PROTO_PERF_FACTOR_REMOTE_CPU,
     UCP_PROTO_PERF_FACTOR_LOCAL_TL,
     UCP_PROTO_PERF_FACTOR_REMOTE_TL,
-    UCP_PROTO_PERF_FACTOR_LATENCY,
-    UCP_PROTO_PERF_FACTOR_SINGLE, /* For compatibility; to remove */
-    UCP_PROTO_PERF_FACTOR_MULTI,  /* For compatibility; to remove */
+    UCP_PROTO_PERF_FACTOR_LOCAL_MTCOPY,
+    UCP_PROTO_PERF_FACTOR_REMOTE_MTCOPY,
```

Comment:
Both options look OK for me, @yosefe WDYT?

Reply:
the operation itself is copy between memory types
MEMTYPE_OP in protov2 refers to put/get opcode to pass to UCT

Reply:
- In our arch it is a specific operation on memtype EP, in theory it may be something different than just copy (maybe in the future)
- MT is very well known acronym for multi-threading, would be good to distinguish memtype from that

Reply:
it's better than MT_COPY, but [M|MEM]TYPE_OP still looks better to me

> Maybe MTYPE_COPY?

Reply:
MTYPE_COPY is better IMO

Reply:
@brminich are you OK with MTYPE_COPY as a compromise?

Reply:
ok

Reply:
Done